src/Scripts/Ecs/Systems/ChasePlayerSystem.gd

```diff
@@ -25,10 +25,12 @@ func on_turn_end():
 		if distance <= entity.get("SightComponent").sight_radius:
 			
 			var direction = entity.position.direction_to(_player.position)
+			var mod = Vector2()
```

Comment:
This should really be called `diff` instead of `mod`. Mod makes me think "modulo".